src/courses/mappers/02.md

```diff
@@ -0,0 +1,128 @@
+---
+order: 2
+next: 03.md
+title: What Is OHDF?
+author: Charles Hu
+---
+
+## What is OHDF?
+
+OASIS Heimdall Data Format (OHDF) is a security data format used to normalize generated data exports from various security tools into a single common data format usable by the Security Automation Framework (SAF) tool suite. The format is defined by the [OHDF schema](https://saf.mitre.org/framework/normalize/ohdf-schema) and its goal is to provide a simple and intuitive means for representing security validation profiles, controls, and results.
```

Comment:
change "security validation" to "security data" throughout. sometimes our data won't be "validations", we might process event log data, which is security relevant but not a validation "scan".


change: for representing security validation profiles, controls, and results.
to: for represented security data context ("profiles"), requirements ("controls"), and measures ("results").

Reply:
Addressed.